WCF/Version.props

```diff
@@ -6,14 +6,14 @@
       Update for every public release. 
     -->
     <SemanticVersionMajor>0</SemanticVersionMajor>
-    <SemanticVersionMinor>25</SemanticVersionMinor>
+    <SemanticVersionMinor>26</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
 
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2016-03-17</SemanticVersionDate>
+    <SemanticVersionDate>2016-06-21</SemanticVersionDate>
```

Comment:
Yes, when going from beta-* to stable release you do not want to change it but when incrementing a version number you do want to change it so this is correct.